src/components/pages/ProblemDetailedPage/index.tsx

```diff
@@ -45,15 +52,45 @@ const ProblemDetailedPage: FC = () => {
   const formattedDate =
     problemDetails.createdAt && formatDate(problemDetails.createdAt);
 
+  const printRef = useRef<HTMLDivElement>(null);
+
+  const handleDownloadPdf = async () => {
+    const element = printRef.current as HTMLDivElement;
+
+    const canvas = await html2canvas(element);
+    const imgData = canvas.toDataURL("image/png");
+    const pdf = new jsPDF("p", "mm");
+
+    let position = 0;
+
+    const imgProperties = pdf.getImageProperties(imgData);
+    const pdfWidth = pdf.internal.pageSize.getWidth() - 20;
+    const pageHeight = 295;
```

Comment:
Where is this number taken from? :)

Reply:
This is  more or less the hight of A4 format

Reply:
I thought there is correlation as 297 is A4 height ;)

Reply:
I can change it to exact a4 height in our case those 3 points doesn't change a thing.

Reply:
No need to ;)